app/index.html

```diff
@@ -1,6 +1,7 @@
 <!DOCTYPE html>
 <html>
 <head>
+  <script src="https://maps.googleapis.com/maps/api/js" type="text/javascript"></script>
```

Comment:
I tried earlier without including this script but it results in Content Security Policy violation